crates/core/tedge_agent/src/file_transfer_server/mod.rs

```diff
@@ -1,4 +1,6 @@
+//TODO Rename this module to tedge_server
 pub mod actor;
 pub mod error;
+mod file_transfer_controller;
```

Comment:
Why not simply `file_transfer`?

Reply:
It actually started with `entity_store_controller`, as I didn't want to name it `entity_store` as the actual entity store module is also named that way. And I wanted to differentiate this from that and hence ended up adding the `controller` suffix. I've renamed it as you suggested.

But, now I'm wondering if naming them `file_transfer_service` and `entity_store_service` would have been clearer/better.